### PR TITLE
[WIP] Migrate issue_query to Clickhouse

### PR DESCRIPTION
**Pull request by izaitsevfb**
Query tested in CH Cloud UI, **not sure how to test integration locally**.

### CODE REVIEW

**Comment by clee2000** [comment]
> unsure how to test locally

Since it's used in multiple places (ex Dr. CI) this one is going to be a bit hard to test

**Review by clee2000 — APPROVED** [review]

**Review by huydhn — APPROVED** [review]
LGTM!

File: torchci/clickhouse_queries/issue_query/query.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    issue.number,
+    issue.title,
+    issue.html_url,
+    issue.state,
+    issue.body,
+    issue.updated_at,
+    issue.author_association
+FROM
+    issues AS issue
+    ARRAY JOIN issue.labels AS labels
```
Comment (huydhn) [inline torchci/clickhouse_queries/issue_query/query.sql:11]: TIL https://clickhouse.com/docs/en/sql-reference/statements/select/array-join :)

**Comment by clee2000** [comment]
Completed in https://github.com/pytorch/test-infra/pull/5634

**Pull request closed without merging**